adapters/eea-direct.js

```diff
@@ -0,0 +1,265 @@
+'use strict';
+import { acceptableParameters, convertUnits } from '../lib/utils';
```

Comment:
Please add a line after `'use strict';`, my only reasoning for this is because I'm overly uptight about some things :)